src/packages/solid/components/Column/Column.styles.ts

```diff
@@ -14,17 +14,52 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+import type { NodeStyles } from '@lightningjs/solid';
 import theme from 'theme';
+import type { Tone } from 'types';
+import type { ComponentStyleConfig, NodeStyleSet } from '../../types/types.js';
+import { makeComponentStyles } from '../../utils/index.js';
 
-const styles = {
-  Column: {
+export interface ColumnStyle {
+  tone: Tone;
+  Container: NodeStyleSet<{ padding: number[] }>;
+}
+
+type ColumnStyleProperties = {
+  display?: NodeStyles['display'];
+  justifyContent?: NodeStyles['justifyContent'];
+  flexDirection?: NodeStyles['flexDirection'];
+};
+
+type ColumnConfig = ComponentStyleConfig<ColumnStyleProperties>;
+
+/* @ts-expect-error next-line themes are supplied by client applications so this setup is necessary */
+const { Column: { styles: themeStyles, tone } = { styles: {}, tone: 'neutral' } } = theme?.componentConfig;
+
+const container: ColumnConfig = {
+  themeKeys: {
+    display: 'display',
+    justifyContent: 'justifyContent',
+    flexDirection: 'flexDirection'
```

Comment:
lets limit the theme-able values to the properties found here https://github.com/rdkcentral/Lightning-UI-Components/blob/develop/packages/%40lightningjs/ui-components/src/components/Column/Column.styles.js

Reply:
and perhaps add gap, that feels like it should be theme-able